test/unit/data/briefcase.js

```diff
@@ -352,6 +353,59 @@ Candace,2,three,three/children/child[1]
     });
   });
 
+  it('should output all rows', (done) => {
+    const formXml = `
+      <?xml version="1.0"?>
+      <h:html xmlns="http://www.w3.org/2002/xforms" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:jr="http://openrosa.org/javarosa">
+        <h:head>
+          <model>
+            <instance>
+              <data id="singlerepeat">
+                <orx:meta>
+                  <orx:instanceID/>
+                </orx:meta>
+                <name/>
+                <age/>
+                <children>
+                  <child>
+                    <name/>
+                    <age/>
+                  </child>
+                </children>
+              </data>
+            </instance>
+            <bind nodeset="/data/orx:meta/orx:instanceID" preload="uid" type="string"/>
+            <bind nodeset="/data/name" type="string"/>
+            <bind nodeset="/data/children/child/name" type="string"/>
+          </model>
+        </h:head>
+        <h:body>
+          <input ref="/data/name">
+            <label>What is your name?</label>
+          </input>
+          <group ref="/data/children">
+            <label>Child</label>
+            <repeat nodeset="/data/children/child">
+              <input ref="/data/children/child/name">
+                <label>What is the child's name?</label>
+              </input>
+            </repeat>
+          </group>
+        </h:body>
+      </h:html>`;
+
+    const inStream = streamTest.fromObjects(
+      (new Array(50)).fill(null).map(_ => instance(uuid(), `<orx:meta><orx:instanceID>${uuid()}</orx:instanceID></orx:meta><name>${uuid()}</name><children><child><name>${uuid()}</name></child></children>`)));
```

Comment:
This test passes without the change. Is that deliberate? It's unlikely that we'd have the exactly same regression so maybe it's ok. I'd tend to do 127 submissions, though, which does fail.

Reply:
yeah i didn't think it mattered but if 127 verifies the change let's do it.